docsrc/contributors/fx_converters.rst

```diff
@@ -0,0 +1,211 @@
+.. _dynamo_conversion:
+
+Dynamo Converters
+==================
+The dynamo converter library in Torch-TensorRT is located in ``TensorRT/py/torch_tensorrt/dynamo/conversion``.
+
+
+
+Steps
+==================
+
+Operation Set
+-------------------
+The converters in dynamo are produced by ``aten_trace`` and falls under ``aten_ops_converters`` ( FX earlier had ``acc_ops_converters``, ``aten_ops_converters`` or ``nn_ops_converters`` depending on the trace through which it was produced).  The converters are registered using  ``dynamo_tensorrt_converter`` for dynamo. The function decorated
```

Comment:
Since only `torch.export` uses `aten_trace`, consider rewording as:

> Our Dynamo paths use a set of PyTorch operators called ATen, which are produced by tracers which translate operators into this core set.